docs/spec_draft.md

```diff
@@ -550,6 +551,64 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.iota
+
+### Semantics
+Fills a `result` array with values in increasing order starting from zero along
+the `iota_dimension` dimension.
+
+Note: For integers, if the dimension size is larger than what the element type's
```

Comment:
Can we have an signed/unsigned underflow with values increasing starting from zero?

We can drop the `Note:` as we are not using anything like that for such special cases for other ops.

Reply:
Since the numbers are incrementing, it is not possible to have an underflow. Will drop the `Note:` part.